src/main/java/org/apache/commons/io/input/RandomAccessFileInputStream.java

```diff
@@ -73,12 +73,17 @@ public static class Builder extends AbstractStreamBuilder<RandomAccessFileInputS
         @SuppressWarnings("resource") // Caller closes depending on settings
         @Override
         public RandomAccessFileInputStream get() throws IOException {
+            if (randomAccessFile == null && getOrigin() == null) {
```

Comment:
I wonder if in this case NullPointerException is correct and we should change the doc to match the current behavior instead?